cmd/decodeasm/main.go

```diff
@@ -54,9 +54,7 @@ func main() {
 	cp := capstone.NewCapstone(arch, mode)
 	defer cp.Close()
 	res := cp.Decode(opcode)
-	if len(res) != 1 {
-		fmt.Fprintf(flag.CommandLine.Output(), "expected 1 instruction, got %d\n", len(res))
-		os.Exit(1)
+	for _, d := range res {
```

Comment:
Makes sense, can you add a check for len == 0 with an error since it is usually a decode failure?

Reply:
done